src/torrent.ts

```diff
@@ -64,35 +66,43 @@ export async function snatch(
 		});
 	} catch (e) {
 		if (e.name === "AbortError") {
-			logger.error(`snatch timed out from ${tracker}: ${url}`);
+			logger.error(
+				`Snatch timed out from ${tracker} for ${candidate.name}`,
+			);
+			logger.debug(`${candidate.name}: ${url}`);
 			return resultOfErr(SnatchError.ABORTED);
 		} else if (isMagnetRedirectError(e)) {
 			logger.error(
-				`Unsupported: magnet link detected at ${tracker}: ${url}`,
+				`Unsupported: magnet link detected from ${tracker} for ${candidate.name}`,
 			);
+			logger.debug(`${candidate.name}: ${url}`);
 			return resultOfErr(SnatchError.MAGNET_LINK);
 		}
-		logger.error(`failed to access ${tracker}: ${url}`);
-		logger.debug(e);
+		logger.error(`Failed to access ${tracker} for ${candidate.name}`);
+		logger.debug(`${candidate.name}: ${url} - ${e}`);
 		return resultOfErr(SnatchError.UNKNOWN_ERROR);
 	}
 
 	if (response.status === 429) {
 		return resultOfErr(SnatchError.RATE_LIMITED);
 	} else if (!response.ok) {
 		logger.error(
-			`error downloading torrent from ${tracker} at ${url}: ${response.status} ${response.statusText}`,
+			`Error downloading torrent from ${tracker} for ${candidate.name}: ${response.status} ${response.statusText}`,
+		);
+		logger.debug(
+			`${candidate.name}: ${url} - Response: ${await response.text()}`,
```

Comment:
I don't love the idea of logging the entire response text - they tend to be really large and murder log file usability IME. A common pattern I've adopted is logging the first 1000 characters, would that work here?